### DISCUSSION TITLE
Change password

### DISCUSSION THREAD

**Discussion by ericemmrich** [neutrinolabs/xrdp#2222] [Q&A] [answered]
Hey there,

when I force a password change using `chage -d 0 USERNAME` and try to re-login afterwards using xrdp, I get the "login failed for display 0" error.

Is there any way that I can get to the system's lock screen at least in order to change my password?

Many thanks in advance.

**Comment by matt335672** [comment, marked as answer]
Not at the moment - we're working on it though. See #1961

**Reply by ericemmrich** [reply]
Many thanks for the quick reply. Already looking forward to the redesigned authentication architecture then 👍
